app/helpers/session_timeout_warning_helper.rb

```diff
@@ -11,6 +11,13 @@ def warning
     (Figaro.env.session_timeout_warning_seconds || 30).to_i
   end
 
+  def timeout_refresh_url
+    URI(request.original_url).tap do |url|
+      query = Rack::Utils.parse_nested_query(url.query).with_indifferent_access
+      url.query = query.merge(timeout: true).to_query
+    end.to_s.html_safe # rubocop:disable Rails/OutputSafety
+  end
```

Comment:
re: html escaping

I believe this is the right way. Without this, `&` in URLs turn into `&amp;` which is incorrect (Rails is escaping the URLs for HTML content as opposed to JS literal values).

This URL is used in Javascript in `_expire_session.js.erb` as part of Javascript, so Rails is incorrectly HTML escaping it when it's inside of JS. The `j` helper guards against malicious URLs evaluating arbitrary code in JS (ex `key='; alert()'` or something).

re: excluding pages: I'll push a commit to remove the message from the `/start` and `/privacy` pages